core/src/main/java/org/apache/accumulo/core/spi/metrics/MeterRegistryFactory.java

```diff
@@ -69,4 +78,39 @@ interface InitParameters {
    * @return a Micrometer registry that will be added to the metrics configuration.
    */
   MeterRegistry create(final InitParameters params);
+
```

Comment:
I think this method can just be moved into MetricsInfoImpl. The spi package is for interfaces that define the contracts for user-provided server-side plugins. Putting the method here doesn't really make sense because it's an implementation. If you wanted users to be able to provide their own implementation, then you could put a method declaration here. But, I'm not sure that we need that.